org.freedesktop.Sdk.json.in

```diff
@@ -2208,6 +2208,31 @@
                  }
             ]
         },
+        {
+            "name": "gstreamer-libav",
+            "build-options" : {
+                "arch" : {
+                    "i386" : {
+                        "config-opts" : [
+                            "--build=i586-unknown-linux-gnu"
```

Comment:
Dozens of packages have been building without these, don't think they are required?

Reply:
Not sure what those are for, but follow what seems to be done for all other GStreamer plugins actually. Shoulds we just try without?

Reply:
It is probably harmless.